app/(tabs)/index.tsx

```diff
@@ -1,11 +1,74 @@
-import { View, Text } from "react-native";
+import { View, Text, StyleSheet, ScrollView } from "react-native";
+import ConfirmationCard from "../components/confirmationCard";
+import AppointmentCard from "../components/appointmentCard";
 
 const StartPage = () => {
     return (
-        <View>
-            <Text>Home Page</Text>
-        </View>
+        <ScrollView>
```

Comment:
Could you make the background color of the entire page white? In the designs it's white and there will be a nicer contrast, especially with the Appointment card.